internal/controllers/nmc_reconciler.go

```diff
@@ -431,14 +432,14 @@ func (h *nmcReconcilerHelperImpl) SyncStatus(ctx context.Context, nmcObj *kmmv1b
 		if status == nil {
 			status = &kmmv1beta1.NodeModuleStatus{
 				ModuleItem: kmmv1beta1.ModuleItem{
-					ImageRepoSecret:    nil,
-					Name:               modName,
-					Namespace:          modNamespace,
-					ServiceAccountName: "",
+					Name:      modName,
```

Comment:
we need to remember to fix it in the upstream also

Reply:
https://github.com/kubernetes-sigs/kernel-module-management/pull/573